party_time/party_time.lua

```diff
@@ -171,12 +190,64 @@ pt:register('auto_decline', auto_decline_enable, '<enabled:lookup_boolean>')
 
 
 local default_handler = function(option)
-    options.default = option
+    optio
+    ns.default = option
     settings.save()
 end
 
 pt:register('default', default_handler, '<option:lookup_option>')
 
+
+local kick = function(...)
+    for _, name in pairs({...}) do
+        command.input('/pcmd kick ' .. name)
+    end
+end
+
+pt:register('kick', kick, '{name}*')
+pt:register('k', kick, '{name}*')
+
+
+local leader = function(name)
+    if party.alliance.party_1_leader_id == player.id then
+        if name then
+            --TODO handle if name is in another parrty
+            command.input('/pcmd leader ' .. name)
+        else
+            unhandled_leader = true
+        end
+    else
+        print('You are not the party leader.')
+    end
+end
+
+pt:register('leader', leader, '[name:string(%a+)]')
+pt:register('ldr', leader, '[name:string(%a+)]')
+
+
+local looter = function(name)
+    command.input('/pcmd looter ' .. name)
+end
+
+pt:register('looter', looter, '[name:string(%a+)]')
+pt:register('ltr', looter, '[name:string(%a+)]')
+
+
+local breakup = function(completely)
+    if completely then
+        command.input('/acmd breakup')
+        coroutine.sleep(1)
+        command.input('/pcmd breakup')
+    else
+        local in_alliance = (party.alliance.party_2_leader_id or party.alliance.party_3_leader_id)
+        local cmd = in_alliance and "/acmd" or "/pcmd"
```

Comment:
Mixing of double and single quotes. This happens throughout the file, not gonna mention it again, but should stick to single quotes.